code/game/response_team.dm

```diff
@@ -51,7 +51,7 @@ var/ert_request_answered = 0
 		to_chat(src, "<span class='warning'>This role is not yet available to you. You need to wait another [player_age_check] days.</span>")
 		return 0
 
-	if(has_enabled_antagHUD == 1 && config.antag_hud_restricted)
+	if(canPosses(src))
```

Comment:
Re-thinking this, yes, its definitely wrong.

Before its evaluating to true and stopping you.

With your new "can possess", it is evaluating false and not triggering this if.

These should be `!canPossess(src)`

Reply:
And on that note also, you've typo'd it. Its `Possess` not `Posses`